test/runtest/src/runtest.c

```diff
@@ -165,8 +165,10 @@ runtest_log_result(const char *msg, bool passed)
     int m_len;
     int len;
 
-    /* str length of {"k":"","n":"","s":"","m":"","r":1}<token> */
-    len = 35 + strlen(runtest_token);
+    /* str length of {"k":"","n":"","s":"","m":"","r":1}<token> plus three
+     * null-terminators.
+     */
+    len = sizeof 38 + strlen(runtest_token);
```

Comment:
I guess 38 is 35+3 ("plus three null-terminators"), but what 'sizeof` does here? if this is correct, then I guess it should be written in less cryptic way :)

Reply:
Jeeze... no that is not correct :).  That is a leftover from an earlier fix that I didn't fully remove.  I have fixed it (removed `sizeof`).